packages/bsky/src/services/feed/index.ts

```diff
@@ -343,12 +343,10 @@ export class FeedService {
       repostCount: post.repostCount,
       likeCount: post.likeCount,
       indexedAt: post.indexedAt,
-      viewer: post.viewer
-        ? {
-            repost: post.requesterRepost ?? undefined,
-            like: post.requesterLike ?? undefined,
-          }
-        : undefined,
+      viewer: {
```

Comment:
was there a reason for these changes? seems like it was always getting set to undefined